src/pages/index.js

```diff
@@ -107,18 +134,86 @@ profileAddButton.addEventListener("click", function () {
 
 /* Functions */
 
+function handleAvatarUpdate({ link }) {
+  return api
+    .updateAvatar({ avatar: link })
+    .then((res) => {
+      userInfo.setUserInfo({
+        title: res.name,
+        description: res.about,
+        avatar: res.avatar,
+      });
+      setAvatarPopup.close();
+    })
+    .catch((err) => console.err(`Error updating avatar: ${err}`));
```

Comment:
`catch` is not needed here because you have it inside `PopupWithForm` (it should be only at the very end of the request chain)